go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -360,6 +360,12 @@ func (c *addressManager) nodePrimaryAddrChanged() (bool, error) {
 
 // updateOVNEncapIP updates encap IP to OVS when the node primary IP changed.
 func (c *addressManager) updateOVNEncapIPAndReconnect() {
+	// Skip updating the encapsulation in the DPU mode, since it can have two instances
+	// of ovnkube which refer to different hosts
+	if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		return
+	}
```

Comment:
Right now the DPU needs "external_ids:ovn-encap-ip" to be set.

Reply:
Ack thanks for pointer.
I saw it while rebasing and will account for it in the incremental